### PR TITLE
Added documentation for Compass

### PR DESCRIPTION
**Pull request by NishithP2004**
* **Please check if the PR fulfills these requirements**
- [x] The commit message follows our guidelines
- [ ] Tests for the changes have been added (for bug fixes / features)
- [x] Docs have been added / updated (for bug fixes / features)

* **What kind of change does this PR introduce?** (Bug fix, feature, docs update, ...)
- [ ] Bug fix
- [ ] Feature implementation
- [x] Doc updates

* **Related Issue**

#60 

* **What changes have you introduced?**
Hi, I have added documentation for Compass in the form of an experiment.

* **Does this PR introduce a breaking change?**

* **Preview / Steps to verify your work**:
[Preview link](https://nishithp.live/PSLab-documentation)

### CODE REVIEW

**Comment by NishithP2004** [comment]
Sir, I'm unable to create a preview link.

**Comment by NishithP2004** [comment]
@cweitat Please review

**Comment by cweitat** [comment]
you can create github pages from this repo and share your github pages link

**Comment by NishithP2004** [comment]
> you can create github pages from this repo and share your github pages link

I tried building GitHub pages with the master branch but it isn’t working.

**Comment by NishithP2004** [comment]
@cweitat Could you please assist me with the former.

File: tutorials/compass.md [hunk 1]
```diff
@@ -1,3 +1,49 @@
+# Compass
+
+## What is a Compass?
+A compass is an instrument used for navigation and orientation that shows direction relative to the geographic cardinal directions. Usually, a diagram called a compass rose shows the directions north, south, east, and west on the compass face as abbreviated initials.
+______________
+## EXPERIMENT FOR PSLAB COMPASS
+
+### Goal of the experiment :-
```
Comment (cweitat) [inline tutorials/compass.md]: please remove the :- . same for other headings

File: tutorials/compass.md [hunk 2]
```diff
@@ -1,3 +1,49 @@
...
+<table>
+     <tr>
+         <td><img src="/_static/img_compass_2.PNG"></td>
+         <td><img src="/_static/img_compass_1.PNG"></td>
+     </tr>
+ </table>
+
+* Select the compass option
+* Once the compass opens start rotating the phone 360 degrees in order to calibrate the compass.
+* Now,rotate your phone in any cardinal direction.
```
Comment (cweitat) [inline tutorials/compass.md]: missing space after comma

File: tutorials/compass.md [hunk 3]
```diff
@@ -1,3 +1,49 @@
...
+
+![E52CF8D6-ECFB-4D20-B10D-4E357BD73269](https://user-images.githubusercontent.com/34577844/71526783-6f294000-28fe-11ea-8dbd-527448eed584.png)
```
Comment (cweitat) [inline tutorials/compass.md]: reduce height of this image. too big

File: tutorials/compass.md [hunk 4]
```diff
@@ -1,3 +1,49 @@
...
+
+### Observation:-
+
+* On rotating the phone from one direction to another, the Compass will start rotating denoting the particular direction (N,S,E & W or Subcardinal directions).
+* Alternatively , you may also select the  axes parallel to the ground like **X axis , Y axis or Z axis**.
+* You may even record the interaction using the in-built record option present in the top right corner of the Compass.
+* To view the recorded interactions, you just have to go to the **Logged Data** option present in the menu bar.
+
+![40E1BC7D-49CD-45C1-9243-76C27F346911](https://user-images.githubusercontent.com/34577844/71526785-6f294000-28fe-11ea-9ed9-3c4ca06c1da4.png)
```
Comment (cweitat) [inline tutorials/compass.md]: same for this

File: tutorials/compass.md (hunk 3 above)
Comment (cweitat) [reply tutorials/compass.md]: all the images are too big

File: tutorials/compass.md (hunk 4 above)
Comment (cweitat) [reply tutorials/compass.md]: stlil some missing space for numbers at the bottom

File: tutorials/compass.md (hunk 3 above)
Comment (NishithP2004) [reply tutorials/compass.md]: Could you suggest a recommended  ‘’W x H”....So that I can resize the images accordingly.

**Comment by NishithP2004** [comment]
@cweitat Please review...Changes are ✅.

**Comment by NishithP2004** [comment]
Changes have been made.
Please review @cweitat @hpdang

**Comment by aryanguls** [comment]
@NishithP2004 @mariobehling The second experiment about whether the compass uses magnets or not was added by me to the PSLab docs repository as the name suggests hence shouldnt I be the one to be able to merge it? The doc is names Aryanguls_experiment.md.

**Pull request merged**